api/src/db/migrations/versions/2024_02_21_remove_dms_exceptions_table.py

```diff
@@ -0,0 +1,37 @@
+"""remove DMS exceptions table
+
+Revision ID: 5d58c38f2cac
+Revises: 86edfcb92a66
+Create Date: 2024-02-21 13:44:50.932948
+
+"""
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "5d58c38f2cac"
+down_revision = "86edfcb92a66"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
```

Comment:
I literally just generated an empty migration file and reversed the upgrade/downgrade commands from the prior migration